cmd/podman/varlink/io.podman.varlink

```diff
@@ -804,6 +820,50 @@ method TopPod() -> (notimplemented: NotImplemented)
 # ~~~
 method GetPodStats(name: string) -> (pod: string, containers: []ContainerStats)
 
+# ImageExists talks a full or partial image ID or name and returns an int as to whether
+# the image exists in local storage. An int result of 0 means the image does exist in
+# local storage; whereas 1 indicates the image does not exists in local storage.
+method ImageExists(name: string) -> (exists: int)
+
+# ContainerExists takes a full or partial container ID or name and returns an int as to
+# whether the container exists in local storage.  A result of 0 means the container does
+# exists; whereas a result of 1 means it could not be found.
+method ContainerExists(name: string) -> (exists: int)
+
+# ContainerCheckPoint performs a checkpopint on a container by its name or full/partial container
+# ID.  On successful checkpoint, the id of the checkpointed container is returned.
+method ContainerCheckpoint(name: string, keep: bool, leaveRunning: bool, tcpEstablished: bool) -> (id: string)
+
+# ContainerRestore restores a container that has been checkpointed.  The container to be restored can
+# be identified by its name or full/partial container ID.  A successful restore will result in the return
+# of the container's ID.
+method ContainerRestore(name: string, keep: bool, tcpEstablished: bool) -> (id: string)
+
+# ContainerRefresh resets the state of all containers to handle database changes after an upgrade. All
+# running containers will be restarted.
+method ContainerRefresh() -> ()
+
+# ContainerCleanup removes mount points and network stacks for a container.  The input name
+# may be a container name or a full or partial containter ID.
+method ContainerCleanup(name: string) -> ()
+
+# ContainerRunlabel runs executes a command as described by a given container image label.
+method ContainerRunlabel(runlabel: Runlabel) -> ()
+
+# ListContainerMounts gathers all the mounted container mount points and returns them as an array
+# of strings
+method ListContainerMounts() -> (mounts: []string)
+
+# MountContainer mounts a container by name or full/partial ID.  Upon a successful mount, the destination
+# mount is returned as a string.
+method MountContainer(name: string) -> (path: string)
```

Comment:
If it's on a remote system, they can't do the scan without using our API to do it (or manually SSHing into the box in question, at which point, why not just use podman on that host?). If it goes through our API we can handle the mount/unmount ourselves.

Reply:
Sure, but the API isn't just for python, pypodman, or remote use.  So someone could be developing local plugins and in other languages of course.  Perhaps we agree to not expose this with our python library?

Reply:
Agree. If we're doing that, though, I think we drop the show mounts endpoint? I don't see meaning in that outside of the Podman CLI

Reply:
Id like to keep it; it is easy enough to expose.  And again, we dont rightly know outside the python libraries what folks will do with it yet; if we dont want to expose it beyond the API, then we can just have @jwhonce not expose it

Reply:
Ok I see @baude Point, that doing podman mount from a `import podman` makes some sense.  Also this would allow cockpit to do it from nodejs.  If we run this over SSH, then we should just block the call as not supported,